### PR TITLE
added the docstring for describe_num_var

### LINKED ISSUE #6: Skeleton Function 1: Describe numerical values
- [ ] create skeleton of function to explore numeric variables with doc string 

ideas include density/violin and report min/max/mean

### PR DESCRIPTION
**Pull request by marvinmin**
closes #6

### CODE REVIEW

**Review by sweber15 — APPROVED** [review]
Looks good!

File: eda_analysis/eda_analysis.py [hunk 1]
```diff
@@ -54,9 +54,40 @@ def describe_cat_var(dataframe,cat_vars):
...
+        The dataframe to be inspected.
+    num_vars: list
+        A list of character strings of the names of the numeric variables.
+    
+    Returns:
+    --------
+    dataframe: pandas.DataFrame
+        statistical summary of the numeric variables
+    altair
+        a grid of altair plot containing all histograms
```
Comment (sgauravm) [inline eda_analysis/eda_analysis.py]: You can't return two objects. Return a tuple of the two . Like tuple(pandas.DataFrame, altair)

File: eda_analysis/eda_analysis.py [hunk 2]
```diff
@@ -54,9 +54,40 @@ def describe_cat_var(dataframe,cat_vars):
...
+    >>> cat_vars = ['type']
     >>> describe_cat_variable(X,cat_vars)
        
     """
     # Code 
     
+def describe_num_var(dataframe, num_vars):
+    """ 
+    This function takes data frame and numeric variable names and provides 
+    statistical summary of the numeric variables for a dataframe.
```
Comment (sgauravm) [inline eda_analysis/eda_analysis.py:66]: Change data frame to dataframe

File: eda_analysis/eda_analysis.py (hunk 1 above)
Comment (marvinmin) [reply eda_analysis/eda_analysis.py]: I changed it. Is it now correct?

File: eda_analysis/eda_analysis.py (hunk 2 above)
Comment (marvinmin) [reply eda_analysis/eda_analysis.py:66]: Changed. Thank you!

File: eda_analysis/eda_analysis.py [hunk 3]
```diff
@@ -75,10 +75,11 @@ def describe_num_var(dataframe, num_vars):
     
     Returns:
     --------
-    dataframe: pandas.DataFrame
-        statistical summary of the numeric variables
-    altair
-        a grid of altair plot containing all histograms
+    tuple(pandas.DataFrame, altair)
+        dataframe: pandas.DataFrame
```
Comment (sgauravm) [inline eda_analysis/eda_analysis.py]: only return type should be specified. not the variable name. Remove dataframe and just keep pandas.DataFrame

Comment (marvinmin) [reply eda_analysis/eda_analysis.py]: Got it. Removed. Thank you!

**Review by v5y8 — APPROVED** [review]
looks fine. merging

**Pull request merged**